third_party/coro/coro.c

```diff
@@ -324,7 +329,11 @@ trampoline (int sig)
          ".cfi_offset 30, -16\n"
          "\tmov x0, x20\n"
          "\tblr x19\n"
+# ifdef __APPLE__
```

Comment:
Minor: it looks more convenient to use either `#ifdef` or `#ifndef` in both cases. Feel free to ignore.